### PR TITLE
Added AI-HRI symposium paper

### PR DESCRIPTION
**Pull request by kalebishop**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 source "https://rubygems.org"
 gem 'github-pages'
+
+gem "webrick", "~> 1.8"
```
Comment (alecive) [inline Gemfile:4]: what is this for?

**Review by alecive — CHANGES_REQUESTED** [review]

File: _data/publications.yml [hunk 2]
```diff
@@ -1,3 +1,10 @@
+ - title: "Towards A Natural Language Interface For Flexible Multi-Agent Task Assignment"
+   authors: Jake Brawer, Kayleigh Bishop, Bradley Hayes, Alessandro Roncone
+   year: 2023
+   venue: "AAAI Fall Symposium on Artificial Intelligence for Human-Robot Interaction [AI-HRI]"
+   where: Arlington, VA, USA, Oct 25-27
+   pdf: "2023_Brawer_AI-HRI_task_assignment"
+   subteams: [algorithmic and social hri]
```
Comment (alecive) [inline _data/publications.yml:7]: here you have to add your name and jakes as two other keywords. We may not use them, but better to have it than not

**Review by alecive — CHANGES_REQUESTED** [review]

File: Gemfile (hunk 1 above)
Comment (kalebishop) [reply Gemfile:4]: Ah sorry I can get rid of it, that was something Ruby made me add to host it on my machine

**Comment by kalebishop** [comment]
Fixed the above

**Review by alecive — APPROVED** [review]

**Pull request merged**